ruby-on-rails/docs/gem-tutorials/spree.md

```diff
@@ -0,0 +1,324 @@
+# How to use Spree gem
+
+*Abstract: In this documents, we will present some tricks to deal with spree gem. These tricks will be at nivel database, backoffice, backend, frontend... This will be a good base to start with spree and have a common way to work with this gem.*
```

Comment:
In this document* since it's just one

Reply:
"These tricks will be about the", i don't think that "nivel" is an english word hahaha